android/src/main/java/tech/bam/RNBatchPush/RNBatchModule.java

```diff
@@ -141,6 +142,11 @@ public void optOutAndWipeData(Promise promise) {
     private void sendEvent(ReactContext reactContext,
                            String eventName,
                            @Nullable WritableMap params) {
+
+        if (!reactContext.hasActiveCatalystInstance()) {
+            Log.d(NAME, "React context has no active catalyst instance. Aborting send event.");
```

Comment:
Ideally we would store the event, and send it later once react gets an active catalyst instance
Otherwise you lose the event :/